.eslintrc.js

```diff
@@ -298,8 +304,32 @@ module.exports = {
         'import/no-extraneous-dependencies': 0,
       },
     },
+
+    // TypeScript-specific settings.
+    {
+      files: ['*.ts'],
```

Comment:
Should this have `.tsx` too?

Reply:
Great point, can we possibly add it to https://github.com/netlify/eslint-config-node/blob/main/react_config.js?

Reply:
I think it's fine to add this here, since there could (possibly) be `.tsx` files in projects that don't want to use `react_config` (for example if they do email templating using JSX, but on the backend)
```suggestion
      files: ['*.ts', '*.tsx'],
```

Reply:
Sounds good to me!